custom-commands/hospitality/test/input-files/Test_TurnOnOff_TextInput.json

```diff
@@ -0,0 +1,344 @@
+[
+	{
+		"DialogID": "0",
+		"Description": "Testing 'turn it {OnOff}'",
+		"Skip": false,
+		"Turns": [
+			{
+				"TurnID": 0,
+				"Utterance": "turn it on",
+				"WavFile": "",
+				"ExpectedResponses": [
+					{
+						"type": "message",
+						"text": "Which device would you like to turn on",
+						"speak": "<speak version=\"1.0\" xml:lang=\"EN-US\"><voice name=\"Microsoft Server Speech Text to Speech Voice (en-US, Jessa24kRUS)\">Which device would you like to turn on</voice></speak>",
+						"inputHint": "expectingInput"
+					}
+				],
+				"ExpectedTTSAudioResponseDuration": [
+					2800
+				],
+				"ExpectedResponseLatency": 3800
+			},
+			{
+				"TurnID": 1,
+				"Utterance": "air conditioning",
+				"WavFile": "",
+				"ExpectedResponses": [
+					{
+						"type": "message",
+						"text": "Ok, turning the AC on",
+						"speak": "<speak version=\"1.0\" xml:lang=\"EN-US\"><voice name=\"Microsoft Server Speech Text to Speech Voice (en-US, Jessa24kRUS)\">Ok, turning the AC on</voice></speak>",
+						"inputHint": "acceptingInput"
+					}
+				],
+				"ExpectedTTSAudioResponseDuration": [
+					2800
+				],
+				"ExpectedResponseLatency": 2500
```

Comment:
Any reason you are using different values of response latency? If not, can you use the same value in all tests? try to make it a small number.